pkgs/tools/security/ghidra/extensions/ghidra-firmware-utils/default.nix

```diff
@@ -0,0 +1,24 @@
+{
+  buildGhidraExtension,
+  fetchFromGitHub,
+  lib,
+}:
+buildGhidraExtension rec {
```

Comment:
prefer `let in` to `rec` https://github.com/NixOS/nixpkgs/issues/315337

Reply:
Silly me, `finalAttrs` isn't a thing on non-`mkDerivation` builders.

That said, the `let self = buildGhidraExtension ...; in self` pattern propagated by the other extensions that use `let` seems to be even more of a readability/understandability nightmare, so I'd prefer to just keep it as-is with the `rec` keyword, as there doesn't seem to be an advantage either way.

Reply:
> Silly me, `finalAttrs` isn't a thing on non-`mkDerivation` builders.

This is now rectified in #418706, depending on which PR gets merged first I'll adjust the other one after.